cmd/snap/cmd_services.go

```diff
@@ -184,8 +185,61 @@ func (s *svcLogs) Execute(args []string) error {
 	return nil
 }
 
+type userAndScopeMixin struct {
+	System bool   `long:"system"`
+	User   bool   `long:"user"`
+	Users  string `long:"users" optional:"yes" optional-value:"all"`
```

Comment:
can you remind me what optional-value:"all" does ?

Reply:
Yes, it allows you to use `--users` and the value will default to `=all`. So doing `--users` is equivalent to using `--users=all`

Reply:
Updated this to not allow `--users`